spring-cloud-loadbalancer/src/main/java/org/springframework/cloud/loadbalancer/blocking/client/BlockingLoadBalancerClient.java

```diff
@@ -58,11 +59,19 @@
 
 	private final LoadBalancerProperties properties;
 
+	private LoadBalancerPropertiesFactory propertiesFactory;
+
+	@Deprecated
```

Comment:
Please add javadoc with `@deprecated` tag indicating the suggested replacement.

Reply:
Added